Modules/_ctypes/callproc.c

```diff
@@ -1443,40 +1439,31 @@ copy_com_pointer(PyObject *self, PyObject *args)
 }
 #else
 
-#ifdef HAVE_DYLD_SHARED_CACHE_CONTAINS_PATH
-static PyObject *py_dyld_shared_cache_contains_path(PyObject *self, PyObject *args)
-{
-     PyObject *name, *name2;
-     char *name_str;
-
-     if (__builtin_available(macOS 11.0, iOS 14.0, tvOS 14.0, watchOS 7.0, *)) {
-         int r;
-
-         if (!PyArg_ParseTuple(args, "O", &name))
-             return NULL;
-
-         if (name == Py_None)
-             Py_RETURN_FALSE;
-
-         if (PyUnicode_FSConverter(name, &name2) == 0)
-             return NULL;
-         name_str = PyBytes_AS_STRING(name2);
+#ifndef MS_WIN32
+static PyObject *py_shared_library_is_loadable(PyObject *self, PyObject *args) {
+    PyObject *name, *name2;
+    char *name_str;
+    void* handle;
```

Comment:
C99 is now accepted, so please use intermingled declaration and move this down to line 1458:

```c
    void *handle = dlopen(name_str, RTLD_LAZY);
```